spec/features/admin/unit_price_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+feature '
+    As an admin
+    I want to check the unit price of my products/variants
+' do
+  include AuthenticationHelper
+  include WebHelper
+  
+  let!(:stock_location) { create(:stock_location, backorderable_default: false) }
+  
+  before do
+    allow(OpenFoodNetwork::FeatureToggle).to receive(:enabled?).with(:unit_price, anything) { true }
+  end
+  
+  describe "product", js: true do
+    scenario "creating a new product", :debug do
+      login_as_admin_and_visit spree.admin_products_path
+      click_link 'New Product'
+      select "Weight (kg)", from: 'product_variant_unit_with_scale'
+      fill_in 'product_unit_value_with_description', with: '1'
+      fill_in 'product[price]', with: '1'
+      
+      expect(find("#variant_unit_price", :visible => false).value).to eq '$1.00 / kg'
```

Comment:
Why is this not visible? You should be able to do this:
```suggestion
      expect(page).to have_field("Unit price", with: "$1.00 / kg")
```

I couldn't find the translation for `unit_price` though and I'm wondering if it's missing. I'm currently working on a pull request to avoid all missing translations.

Reply:
Not sure to understand, but yes, I've tried to put translation as much as I can. If i missed something, I'll be happy to fix this. For this particular point (field in New Product form), the key is under: `admin.products.new.unit_price`

Reply:
Yes, I found it: for the 'New Variant' form. Thanks a lot, I'll add it!